src/types/redis_string.cc

```diff
@@ -530,4 +531,156 @@ rocksdb::Status String::CAD(const std::string &user_key, const std::string &valu
   return rocksdb::Status::OK();
 }
 
+rocksdb::Status String::LCS(const std::string &user_key1, const std::string &user_key2, StringLCSArgs args,
+                            StringLCSResult *rst) {
+  if (args.type == StringLCSType::LEN) {
+    *rst = static_cast<uint32_t>(0);
+  } else if (args.type == StringLCSType::IDX) {
+    *rst = StringLCSIdxResult{{}, 0};
+  } else {
+    *rst = std::string{};
+  }
+
+  std::string a;
+  std::string b;
+  std::string ns_key1 = AppendNamespacePrefix(user_key1);
+  std::string ns_key2 = AppendNamespacePrefix(user_key2);
+  MultiLockGuard guard(storage_->GetLockManager(), std::vector{ns_key1, ns_key2});
```

Comment:
I don't think we need a lock since we don't change anything here.

Reply:
Oh, sorry. I have removed it.